sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -160,7 +160,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 [jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
 [logging]: https://github.com/Azure/azure-sdk-for-java/wiki/Logging-with-Azure-SDK
 [OpenTelemetry-quickstart]: https://github.com/open-telemetry/opentelemetry-java
-[OpenTelemetry]: https://github.com/open-telemetry/opentelemetry-java
+[OpenTelemetry]: https://azure.github.io/azure-sdk-for-java/
```

Comment:
Why are we changing the link to the OpenTelemetry project to our own SDK documentation?

Reply:
Sorry not to notice that `OpenTelemetry` and `API reference documentation` use the same link. In addition, the content related to `OpenTelemetry `is not recorded in our own SDK documentation, so the currently specified link of `API reference documentation` is more appropriate and does not need to be modify. What do you think?

Reply:
Hey @hongli19750210, I think we should change the product documentation link [here](https://github.com/Azure/azure-sdk-for-java/pull/21879/files#diff-7e9f8c894b8102b7c480b42764367e4c45472d793a0f52419a278648eb6af687R9) link to reference Opentelemetry documentation.
i.e 
>[Product
documentation][api_documentation]  -> [Product
documentation][OpenTelemetry]